content/pt-br/docs/reference/glossary/kubectl.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Kubectl
+id: kubectl
+date: 2018-04-12
+full_link: /docs/user-guide/kubectl-overview/
+short_description: >
+  Uma ferramenta de linha de comando para se comunicar com um cluster Kubernetes.
+
+aka:
+- kubectl
+tags:
+- tool
+- fundamental
+---
+Ferramenta de linha de comando para se comunicar com o {{< glossary_tooltip text="plano de controle" term_id="control-plane" >}} de um cluster Kubernetes usando a API do Kubernetes.
```

Comment:
"camada de gerenciamento" would be our agreed upon term in Portuguese for control plane.